### ISSUE TITLE
Passing Firebird dpb isc_dpb_no_db_triggers logs a warning

### ISSUE THREAD

**Issue by asfernandes** [FirebirdSQL/jaybird#32]
```java
final Properties properties = new Properties();
properties.setProperty("isc_dpb_no_db_triggers", "1");

connection = (FBConnection) DriverManager.getConnection(
	"jdbc:firebirdsql:embedded:" + databaseName + "?charSet=UTF-8", properties);
```

```txt
jul 03, 2020 11:16:22 PM org.firebirdsql.logging.JulLogger warn
WARNING: Unknown or unsupported parameter with type 72 added to extra database parameters
```

Jaybird 4.0.0

**Comment by mrotteveel** [comment]
Jaybird will log this for any DPB property that is not explicitly supported by Jaybird. I'll map it suppress the warning.

**Comment by mrotteveel** [comment]
Created http://tracker.firebirdsql.org/browse/JDBC-628

**Comment by mrotteveel** [comment]
Fixed for 4.0.1, 3.0.10 and 5
